src/ape_ethereum/multicall/handlers.py

```diff
@@ -87,17 +87,13 @@ def handler(self) -> ContractTransactionHandler:
         if any(call["value"] > 0 for call in self.calls):
             return self.contract.aggregate3Value
 
-        elif any(call["allowFailure"] for call in self.calls):
-            return self.contract.aggregate3
-
-        else:
-            return self.contract.aggregate
+        return self.contract.aggregate3
 
     def add(
         self,
         call: ContractMethodHandler,
         *args,
-        allowFailure: bool = False,
```

Comment:
I had it this way to match the semantics of the contract (similar to `.returnData`), but either way changing it turns this into a breaking change that will have to wait for 0.8

Reply:
reverted this back